src/components/tabs/tab.ts

```diff
@@ -0,0 +1,73 @@
+import { html, LitElement } from 'lit';
+import { themes } from '../../theming/theming-decorator.js';
+import { property, query } from 'lit/decorators.js';
+import { styles } from './themes/light/tab.base.css.js';
+import { styles as bootstrap } from './themes/light/tab.bootstrap.css.js';
+import { styles as fluent } from './themes/light/tab.fluent.css.js';
+import { styles as indigo } from './themes/light/tab.indigo.css.js';
+
+/**
+ * Represents the tab header.
+ *
+ * @element igc-tab
+ *
+ * @slot prefix - Renders content before the default tab header.
+ * @slot - Renders the tab header.
+ * @slot suffix - Renders content after the default tab header.
+ *
+ * @csspart prefix - The prefix wrapper.
+ * @csspart suffix - The suffix wrapper.
+ */
+@themes({ bootstrap, fluent, indigo })
+export default class IgcTabComponent extends LitElement {
+  public static readonly tagName = 'igc-tab';
+
+  public static override styles = styles;
+
+  @query('[part="base"]', true)
+  private tab!: HTMLElement;
+
+  /** The name of the tab panel which will be controlled by the tab. */
+  @property({ type: String })
+  public panel = '';
+
+  /** Determines whether the tab is selected. */
+  @property({ type: Boolean, reflect: true })
+  public selected = false;
+
+  /** Determines whether the tab is disabled. */
+  @property({ type: Boolean, reflect: true })
+  public disabled = false;
+
+  /** Sets focus to the tab. */
+  public override focus(options?: FocusOptions) {
+    this.tab.focus(options);
+  }
+
+  /** Removes focus from the tab. */
+  public override blur() {
+    this.tab.blur();
+  }
+
+  protected override render() {
+    return html`
+      <div
+        part="base"
+        role="tab"
+        aria-disabled=${this.disabled ? 'true' : 'false'}
+        aria-selected=${this.selected ? 'true' : 'false'}
```

Comment:
I don't see `aria-controls` linking to the panel the tab controls, the specs say 'should' but I think it might be needed to properly reflect the active tab when the focus is in the panel or its children.
To that end - @rkaraivanov would it not make more sense to have `id` on the panel instead of `name` as the ARIA link is based on id-s anyway?

Reply:
@damyanpetev 
Fine by me. Feel free to update the specification.

Reply:
I did though it's described as _should_ have.
Reading https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role we might also consider aria-owns list on the tabs themselves it seems as it's described as _must_

Reply:
I did though it's described as _should_ have.
Reading https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role we might also consider aria-owns list on the tabs themselves it seems as it's described as _must_